src/CitationAccessControlHandler.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Drupal\stanford_publication;
+
+use Drupal\Core\Entity\EntityAccessControlHandler;
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Session\AccountInterface;
+use Drupal\Core\Access\AccessResult;
+
+/**
+ * Access controller for the Citation entity.
+ *
+ * @see \Drupal\stanford_publication\Entity\Citation.
+ */
+class CitationAccessControlHandler extends EntityAccessControlHandler {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function checkAccess(EntityInterface $entity, $operation, AccountInterface $account) {
+    /** @var \Drupal\stanford_publication\Entity\CitationInterface $entity */
+
+    switch ($operation) {
+
+      case 'view':
+        return AccessResult::allowed();
```

Comment:
Any reason to add a `view citation entities`?

Reply:
:shrug: I didn't see much need at the moment. it doesn't hurt to have it but it can always be added later too. Meh

Reply:
Can't imagine a situation where you want to prevent view access for citations, but that doesn't mean it may not come up sometime down the road.  I'm with Mike on this one.